tidb-cloud/tidb-cloud-import-local-files.md

```diff
@@ -26,40 +26,40 @@ Currently, this method supports importing one CSV file for one task into either
         >
         > If you have multiple projects, you can switch to the target project in the left navigation pane of the **Clusters** page.
 
-    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
+    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane. Alternatively, you can click **...** at the end of the cluster row and then click **Import**.
```

Comment:
Click **...** at the end of the cluster row and then click **Import**.Alternatively, you can click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.